### DISCUSSION TITLE
Get Invoices

### DISCUSSION THREAD

**Discussion by n1c01a5** [community/community#49312] [Enterprise]
### Select Topic Area

Question

### Body

Hi all,

I subscribed to the Github Enterprise for my organization but in the Billing section I have only the receipts and not the invoices.

So where can I download the invoices ?

**Comment by ettaboyle** [comment]
Hi @n1c01a5 ,

This article from the GitHub Docs goes over everything related to [Managing invoices for your enterprise](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-billing-for-your-github-account/managing-invoices-for-your-enterprise) 

Cheers!

**Reply by n1c01a5** [reply]
I don't have any invoices. See 
![Screenshot 2023-03-08 at 10 25 02](https://user-images.githubusercontent.com/1773356/223675034-96151f24-5a21-4d93-aace-48eb09575b74.png)

Because I just moved to Github Enterprise this org and Github reset my receipts.
